src/test/utils/helpers.js

```diff
@@ -31,6 +31,31 @@ export const generateArticle = async ({ authorId }) => ({
   status: 'active'
 });
 
+export const generateReport = async ({ userId, articleId, violation }) => ({
+  userId,
+  articleId,
+  description: faker.lorem.word(),
+  violation
+});
+
+export const reportArticle = async (articleId, report) => {
+  const created = await BaseRepository.findOrCreate(
+    db.Cases,
+    { articleId },
+    report
+  );
+  return created;
+};
+
+export const createArticle2 = async article => {
+  const created = await BaseRepository.findOrCreate(
+    db.Article,
+    { id: article.id },
+    article
+  );
+  return created;
+};
+
```

Comment:
I think these 3 methods (`generateReport, reportArticle, createArticle2`) are not necessary.
*Reasons*:
1) generateReport and reportArticle are kind of similar or contradicts each other.
2) There is a helper function already on this project to use for creating/generating an article.

Reply:
the helper function you talked about are also used in this same function because this function is not depending on any other test but a standalone test, I want to report an article so I need to create an article by the same article generator and then create the report generator to generate the report.